manifests/crd.yaml

```diff
@@ -1443,6 +1443,11 @@ spec:
                     pd-server:
                       description: PDServerConfig is the configuration for pd server.
                       properties:
+                        metric-storage:
+                          description: MetricStorage is the cluster metric storage.
+                            Currently we use prometheus as metric storage, we may
+                            use PD/TiKV as metric storage later. imported from v3.1.0
```

Comment:
maybe we can use `Available since v3.1.0` to tell the user that this config is available since v3.1.0?

Reply:
edit: I added `TiDB` prefix because we should make it clear that `v3.1.0` is the version of TiDB not tidb-operator, etc.